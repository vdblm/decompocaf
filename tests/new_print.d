int main(){
	Print(3.1415);
	Print(3.14159265);
	Print(575450595.1415);
}