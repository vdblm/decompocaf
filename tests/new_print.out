3.14150000
3.14159274
575450624.00000000
